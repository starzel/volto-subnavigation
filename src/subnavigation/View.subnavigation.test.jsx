import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-intl-redux';
import { MemoryRouter } from 'react-router-dom';
import SubnavigationView from './View';

const mockStore = configureStore();

describe('Breadcrumbs', () => {
  it('renders a breadcrumbs component', () => {
    const store = mockStore({
      intl: {
        locale: 'en',
        messages: {},
      },
    });
    const props = {
      data: {
        depth: 2,
        title: 'Subnavigation',
      },
    };
    const component = renderer.create(
      <Provider store={store}>
        <MemoryRouter>
          <SubnavigationView {...props} />
        </MemoryRouter>
      </Provider>,
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
