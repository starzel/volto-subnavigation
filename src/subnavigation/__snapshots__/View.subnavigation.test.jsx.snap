// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs renders a breadcrumbs component 1`] = `
<div
  className="block subnavigation"
>
  <h3>
    Subnavigation
  </h3>
</div>
`;
